src/headless/components/form/radio-buttons/radio-button/headless-radio-item-props.tsx

```diff
@@ -1,13 +1,14 @@
 import { ChangeEvent } from 'react';
-import { ComponentAnimationProps } from '../../../../headless/components/component-interfaces.ts';
+import { ComponentAnimationProps } from '../../../component-interfaces.ts';
 import { SelectionLabelLocation } from '../../form-generic.interfaces.ts';
 
-export interface RadioItemProps extends ComponentAnimationProps {
+export interface HeadlessRadioItemProps extends ComponentAnimationProps {
```

Comment:
The `ComponentAnimationProps` shouldn't be used here since this is a headless component.